Test_Stack.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+from containers.stack import Stack
+
+class TestStackMethods(unittest.TestCase):
+    def test_initialize_empty_stack(self):
+        newStack = Stack()
+        self.asser
+
+    def test_push(self):
+        newStack = Stack()
+        newStack.push(1)
+        self.assertEqual(newStack, Stack([1]))
```

Comment:
Remove. We aren't implementing comparators, be careful, there are multiple instances of this,